vectorstores/alloydb/vectorestores.go

```diff
@@ -157,15 +157,26 @@ func (vs *VectorStore) SimilaritySearch(ctx context.Context, query string, _ int
 	operator := vs.distanceStrategy.operator()
 	searchFunction := vs.distanceStrategy.similaritySearchFunction()
 
-	columns := append(vs.metadataColumns, vs.contentColumn)
+	columns := []string{}
+	columns = append(columns, vs.contentColumn)
 	if vs.metadataJsonColumn != "" {
 		columns = append(columns, vs.metadataJsonColumn)
 	}
 	columnNames := strings.Join(columns, `, `)
-	whereClause := ""
+	whereQuerys := make([]string, 0)
```

Comment:
@Joaquin-Niembro We are allowing bigger PRs but we need to make sure we are communicating all the functionality changes. This is a huge functionality change that we need to agree on

Reply:
done! just changed it to support strings only!